src/main/java/org/ballerinalang/plugins/idea/runconfig/ui/BallerinaApplicationSettingsEditor.java

```diff
@@ -54,6 +54,7 @@ public BallerinaApplicationSettingsEditor(Project project) {
         myProject = project;
         installRunKindComboBox();
         BallerinaRunUtil.installBallerinaWithMainFileChooser(project, myFileField.getComponent());
+        BallerinaRunUtil.installBallerinaWithMainFileChooser(project, myWorkingDirectoryField.getComponent());
```

Comment:
This is incorrect. We need to be able to select a directory as the current working directory. But this will only show the Ballerina files with **main** function in it and we cannot select a directory as well. So you will have to create a new `FileChooserDescriptor` for this.